docs/pages/enroll-resources/application-access/jwt/elasticsearch.mdx

```diff
@@ -36,6 +36,18 @@ xpack.security.authc.realms.jwt.jwt1:
   allowed_audiences: ["https://elasticsearch.example.com:9200"]
 ```
 
+<Admonition type="tip" title="Elastic Cloud">
+For **hosted deployments** in Elastic Cloud, you can configure the same settings
+by editing your deployment.
+
+Go to "Manage user settings and extensions" for "Elaiscsearch", then add the
+configuration snippet above under "User Settings". You may have to use an order
+number bigger than 1.
+
+Note that configuring JWT authentication is not available for **Serverless**
+projects.
```

Comment:
What is the consequence of this? It isn't clear what the user should do for Serverless.